### PR TITLE
Fix panic when logout of downed Hub server

### PR DESCRIPTION
**Pull request by xenserverarmy**

### CODE REVIEW

**Review by rrati — APPROVED** [review]

**Comment by jayunit100** [comment]
Lgtm

**Review by jayunit100 — APPROVED** [review]

File: common/hubserver.go [hunk 1]
```diff
@@ -128,6 +128,10 @@ func (h *hubServer) Logout() bool {
 	urlStr := fmt.Sprintf("%v", u)
 
 	resp, err := h.client.Get(urlStr)
+	if err != nil {
+		log.Printf("ERROR client.Get %s\n", err)
```
Comment (jayunit100) [inline common/hubserver.go:132]: %+v is probably more general for these sorts of prints iirc as a code convention?

**Review by jayunit100 — APPROVED** [review]

File: common/hubserver.go (hunk 1 above)
Comment (rrati) [reply common/hubserver.go:132]: %v is usually best

**Pull request merged**